src/Components/Restaurants.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import List from '@material-ui/core/List/index'
+import ListItem from '@material-ui/core/ListItem/index';
+import Restaurant from './Restaurant.js';
+
+
+class Restaurants extends Component {
+    render() {
+        const restaurants = this.props.restaurants;
+        console.log(restaurants);
```

Comment:
Maybe you should add `no-console` rule to eslint just not to miss next time

Reply:
removed file